app/reducers/__tests__/adminResourcesPageReducer.spec.js

```diff
@@ -0,0 +1,97 @@
+import { expect } from 'chai';
+
+import { createAction } from 'redux-actions';
+import Immutable from 'seamless-immutable';
+
+import types from 'constants/ActionTypes';
+import adminResourcesPageReducer from 'reducers/adminResourcesPageReducer';
+
+describe('Reducer: adminResourcesPageReducer', () => {
+  describe('initial state', () => {
+    const initialState = adminResourcesPageReducer(undefined, {});
+
+    it('resourceIds should be an empty array', () => {
+      expect(initialState.resourceIds).to.deep.equal([]);
+    });
+  });
+
+  describe('handling actions', () => {
+    describe('API.RESOURCES_GET_SUCCESS', () => {
+      const getResourceSuccess = createAction(
+        types.API.RESOURCES_GET_SUCCESS,
+        payload => payload,
+        (payload, meta) => meta
+      );
+
+      it('adds resources ids from action', () => {
```

Comment:
- [ ] You could separate the case where meta.source is 'adminResourcesPage' and the case where meta.source is something else to own `describe` blocks.